src/helm/benchmark/scenarios/contactcenter_convo_base_scenario.py

```diff
@@ -0,0 +1,59 @@
+"""
+Base scenario for the contact center conversation.
+
+This scenario defines the basic input structure of the conversation
+and shared functions for the contact center conversation scenarios.
+
+The conversation structure is json format with the following fields:
+[
+    {
+        "conversation_name": "cresta-helm-cc-2018-01-12T19:31:31.404000",
+        "body": [
+            {
+                "id": "abcd-1",
+                "text": "Hello?",
+                "speaker_role": "visitor",
+            },
+            {
+                "id": "abcd-2",
+                "text": "Thank you for contacting xxx! My name is Jack. I am here to help you. How can I help you today?",
+                "speaker_role": "agent",
+            }
+        ],
+    },
+    ...
+]
+"""
+
+import json
+from .scenario import Scenario
+
+class ContactCenterConversationScenario(Scenario):
+    """Base scenario for the contact center conversation."""
+    name = "cc_conversation"
+    description = "Base scenario for contact center conversation tasks"
+    tags = ["cc_conversation"]
```

Comment:
You can just delete `name`, `description` and `tags` since this is not a concrete class and can't be instantated.